code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -692,7 +692,6 @@ datum
 			overdose_threshold = 200 // Hyperglycaemic shock
 
 			on_mob_life(var/mob/living/M as mob)
-				M.nutrition += 1*REM

```

Comment:
Won't this make space twinkies and several candies non-nutritive? Not that it really makes sense for them to provide "nutrition", I suppose, but it does remove their actual value as food.